src/DataObjects/QueuedJobDescriptor.php

```diff
@@ -573,6 +582,31 @@ public function getCMSFields()
         return $fields->makeReadonly();
     }
 
+    /**
+     * Generate a Dropdown field with the list of possible job
+     */
+    private function buildJobStatusField(): DropdownField
+    {
+        $statuses = $this->getJobStatusValues();
+        return DropdownField::create(
+            'JobStatus',
+            $this->fieldLabel('JobStatus'),
+            array_combine($statuses ?? [], $statuses ?? [])
+        );
+    }
+
+    /**
+     * Generate a drop down field with a list of possible job types
+     */
+    private function buildJobTypeField(): DropdownField
+    {
+        return DropdownField::create(
+            'JobType',
+            $this->fieldLabel('JobType'),
```

Comment:
The label for this in the summary fields is "Job Type" and it uses `_t(__CLASS__ . '.JOB_TYPE', 'Job Type')` instead of field label - we should probably use that here as well so it's clear they are the same field.